packages/cli/src/entrypoint.ts

```diff
@@ -4,11 +4,22 @@ import { Package } from "./package";
 import { JSONValue } from "./utils";
 import normalizePath from "normalize-path";
 
+export type ExportsConditions = {
+  module?: string;
+  default?: string;
+};
+
+export type ExportsItem = {
+  browser?: ExportsConditions | string;
+  worker?: ExportsConditions | string;
+} & ExportsConditions;
```

Comment:
Those are always "recursive" and can be freely nested anyhow one wants. That makes me think that we shouldn't use their existence as hints for certain build types - it feels like `package.json#exports` should be an artifact created by us and that we should enforce its shape. Otherwise it might get very unwieldy to handle

Reply:
Ok, once we come to an agreement on how to handle dev/prod environments then I'd be happy to reflect that in the types here.